Public/Src/Cache/ContentStore/Distributed/NuCache/EventStreaming/EventHubContentLocationEventStore.cs

```diff
@@ -237,7 +237,9 @@ private async Task ProcessEventsAsync(OperationContext context, List<EventData>
                 await context.PerformOperationAsync(
                     Tracer,
                     () => sendToActionBlockAsync(),
-                    traceOperationStarted: false).TraceIfFailure(context);
+                    traceOperationStarted: false,
+                    extraEndMessage: _ => extraEndMessageForSendToActionBlock(),
+                    caller: "SendToActionBlockAsync").TraceIfFailure(context);

```

Comment:
>SendToActionBlockAsync [](start = 29, length = 22)

`nameof`?

Reply:
There is no function with this name. I decided to add a name that explains what is going on.

---
In reply to: [320375145](https://github.com/microsoft/BuildXL/pull/821#discussion_r320375145) [](ancestors = 320375145)